molecule/default/prepare.yml

```diff
@@ -1,3 +1,9 @@
 ---
+- name: Group hosts by OS distribution
+  hosts: all
+  tasks:
+    - name: Group hosts by OS distribution
+      ansible.builtin.group_by:
+        key: os_{{ ansible_distribution }}
```

Comment:
Will that fix https://github.com/cisagov/ansible-role-admiral/actions/runs/5868373083/job/15911053002?pr=3#step:6:738?

Reply:
No.  I'm not sure what fixes that yet.  I think something broke upstream.

Reply:
You may not even need the stuff in the `prepare.yml` playbook.  I'll let you know about the other thing once it is diagnosed.

Reply:
I don't think commits f6bd659 or 6c3e541 are necessary, since the GH Actions checks for 5315d99 pass.  (I re-ran the failed checks for that commit.)  Can you remove those two commits?

If not then I can rebase and remove them for you, just let me know.

Reply:
Done!